### PR TITLE
Get next u2f counter and add tests for it

### PR DESCRIPTION
**Pull request by tsusanka**
This adds:

- A new message GetNextU2FCounter, which allows the user to get an increased value of the U2F counter.
- Support for python, core and legacy.
- Device test, which sets and gets the counter. Runs against both legacy and core.
- Upgrade tests for both legacy and core.

---

What do you think about the name "GetNextU2FCounter"? Maybe GetIncreasedU2FCounter is better?

---

Closes https://github.com/trezor/trezor-firmware/issues/535.

### CODE REVIEW

File: python/src/trezorlib/cli/trezorctl.py [hunk 1]
```diff
@@ -412,6 +412,12 @@ def set_u2f_counter(connect, counter):
     return device.set_u2f_counter(connect(), counter)
 
 
+@cli.command(help="Get U2F counter.")
+@click.pass_obj
+def get_u2f_counter(connect):
```
Comment (prusnak) [inline python/src/trezorlib/cli/trezorctl.py]: Just a nitpick - shouldn't this follow the suit and be named `get_next_u2f_counter`?

**Comment by prusnak** [comment]
utACK

File: python/src/trezorlib/cli/trezorctl.py (hunk 1 above)
Comment (tsusanka) [reply python/src/trezorlib/cli/trezorctl.py]: It should. Fixed

**Comment by bosomt** [comment]
i have trezorctl v 0.11.5 was this feature implemented in newer version ?

**Comment by tsusanka** [comment]
It is in master currently and not yet released. But I do not think QA needs to test this one.

**Pull request merged**